tests/Test/Disciplina/DB/DSL/SimpleTxDB.hs

```diff
@@ -0,0 +1,172 @@
+module Test.Disciplina.DB.DSL.SimpleTxDB where
+
+import Test.Common
+
+import Crypto.Error (CryptoFailable(..))
+import Disciplina.DB (QueryTx(..), QueryTxs(..), WHERE(..)
+                     ,TxIdEq(..), TxGrade(..)
+                     ,TxsFilterExpr(..), runSimpleTxDBQuery)
+import Disciplina.Educator (PrivateTx(..), EducatorTxMsg(..)
+                           ,StudentTxMsg(..), PrivateTxPayload(..))
+import Disciplina.Crypto (hash, AbstractPK(..), AbstractSK(..), PublicKey, SecretKey)
```

Comment:
I don't really want to import AbstractPK, AbstractSK here but didn't find another way to create PublicKey, SecretKey.